packages/fast-components-styles-msft/src/accent-button/index.ts

```diff
@@ -32,14 +32,14 @@ const styles: ComponentStyles<AccentButtonClassNameContract, DesignSystem> = {
         color: accentForegroundCut,
         fill: accentForegroundCut,
         background: accentFillRest,
-        "&:hover:enabled": {
+        "&:hover:enabled, a&:hover": {
```

Comment:
Per our chat, it looks like these need inclusion of `button__disabled` in some way.

Reply:
can you elaborate here so we have online context? :)

Reply:
The existing selector includes `:enabled` so we don't get hover and active styling on disabled buttons. Since <a> elements don't support the actual `disabled` attribute, we use the class instead. We need the class name in the selector so we don't end up with styles changing on hover and active.

Reply:
Yeah something we caught yesterday. Spent sometime experimenting last night and this morning. Ended up using `:not()` css selector. This allows us to just replace and instance of `:enabled` selector across all buttons.